src/atoms/Space/Space.native.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { View } from 'react-native';
+
+import { getPxScale } from '../../_helpers/theme';
+import automation from '../../_helpers/automation-attributes';
+
+const Space = styled(({ margin, padding, children, ...props }) =>
+  children ? (
+    React.cloneElement(children, props)
+  ) : (
+    <View {...props} {...automation('space-default-view')} />
```

Comment:
Shouldn't render a view by default. It's an enhancer. it takes child and enhances it by adding spaces. We need to add a check that the children you get is just one and only one node  else throw an error. You can do it in propTypes validation. [check this](https://reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child). You can customise the propType validation. [Check this](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes) Scroll to the last 2-3 examples